silverback/_cli.py

```diff
@@ -635,6 +642,14 @@ def update_bot(
     if network:
         click.echo(f"Network:\n  old: {bot.network}\n  new: {network}")
 
+    registry_credentials_id = None
+    if registry_credentials_name:
```

Comment:
So, if you forget to add `--registry-credentials` it will remove them?

Reply:
no

Reply:
So you can't remove credentials once they are set? That's probably not too big a deal, it shouldn't hurt accessing public registries, just making sure we have all the scenarios covered with this (different from create bot)

Reply:
I don't really have a good way to handle this. I'm kicking the can down the road for now.

Reply:
I don't think it will actually cause any issues. It will just unnecessarily log in whenever it pulls that image.

If we come up with a better way of doing it down the road, we can implement it. But I don't see any blocker here.